### DISCUSSION TITLE
Add provider for issues

### DISCUSSION THREAD

**Discussion by joseivanlopez** [agama-project/agama#993] [Ideas]
Issues are retrieved from different parts:

* Installation summary
* Storage page
* Issues page
* ...

For all of them, we have code to get the issues and to subscribe to changes. Adding a new react provider for issues to the list of current providers would simplify the code.

**Comment by dgdavid** [comment]
Closing because the mentioned provider was added ~1month ago as part of https://github.com/openSUSE/agama/pull/1313/commits/fee2c633d994b599dc21088f3cca4a4cc7a8e93f.
